### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by heyvito**
Detalhes, detalhes. ✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,5 +81,5 @@ npm i -g hyperspace-cli
 
 - [ ] App Name dictionary
 - [ ] Clear the desktop saving in a _tempfile
-- [ ] Inject the project files
-- [ ] Storage everthing in the cloud
+- [ ] Inject project files
+- [ ] Store everthing in the cloud
```
Comment (klzns) [inline README.md]:
```diff
- everthing
+ everything
```

**Review by klzns — CHANGES_REQUESTED** [review]
That's great! Can you change that as well?

File: README.md (hunk 1 above)
Comment (heyvito) [reply README.md]: Whoops! Missed that one.

**Review by klzns — APPROVED** [review]

**Comment by klzns** [comment]
Thanks!

**Pull request merged**